src/pages/workspace/WorkspacePeoplePage.js

```diff
@@ -169,17 +183,32 @@ class WorkspacePeoplePage extends React.Component {
     renderItem({
         item,
     }) {
+        const initialDimensions = Dimensions.get('window');
+        const isSmallScreenWidth = initialDimensions.width <= variables.mobileResponsiveWidthBreakpoint;
+        const isMobileDevice = getPlatform() === CONST.PLATFORM.ANDROID || getPlatform() === CONST.PLATFORM.IOS || isSmallScreenWidth;
```

Comment:
`getPlatform()` is a restricted method that should never be used inside a component. The proper way to handle this is to create platform specific files (eg. `.ios.js`, `.android.js`, etc.). 

You could perhaps do this with a new component called `CheckBoxWithTooltip` and moving this logic inside that new component.

I also think that `renderItem()` should be moved to its own file to help with this.

Lastly, there is a higher order component called `withWindowDimensions` that could be useful.

Reply:
Ok!! I will do that!!